packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/update/update.module.js

```diff
@@ -0,0 +1,24 @@
+import angular from 'angular';
+
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import '@ovh-ux/ng-translate-async-loader';
+import ngAtInternet from '@ovh-ux/ng-at-internet';
+
+import component from './update.component';
+import routing from './update.routing';
+
+const moduleName = 'ovhManagerPccDashboardUpdate';
+
+angular
+  .module(moduleName, [
+    ngAtInternet,
+    'ngTranslateAsyncLoader',
```

Comment:
do you need this?

Reply:
Like Marie pointed out, this is required to inject translations :)